src/GridView/CardGrid.js

```diff
@@ -13,6 +13,11 @@ import config from './../config/config';
 import Utils from './../Utils/Utils';
 
 class CardGrid extends ConnectionComponent {
+	constructor(props) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove the unused function parameter "props". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://bpt-lab.org/sonarqube/coding_rules#rule_key=javascript%3AUnusedFunctionArgument)